.babelrc

```diff
@@ -2,8 +2,7 @@
   "presets": [
     ["env", {
       "targets": {
-        "node": "8",
-        "uglify": true
+        "node": "8"
```

Comment:
Why do we target node 8, if we raised the minimum node version to 12?

Reply:
I have no idea. It doesn't even make sense to target Node, as Babel only runs on code for the frontend.

Reply:
I changed the Babel settings to `browsers: defaults`, which equals `> 0.5%, last 2 versions, Firefox ESR, not dead`, which seems more reasonable.
Not sure if that had any effect on bundle size...